%YAML:1.0
image_width: 1280
image_height: 720

k1 : -0.04232352253440998

k2 : -0.0695705588108706

p1 : 0.00036969617368999216

p2 : 0.0055005951703489125

k3 : 0.0

fx : 593.0293023592474

fy : 594.729610749127

cx : 317.2200992770258

cy : 236.15956684611965